include/bluetooth/bluetooth.h

```diff
@@ -775,6 +830,246 @@ int bt_le_ext_adv_get_info(const struct bt_le_ext_adv *adv,
+	 *  This callback notifies the application that the periodic advertising
+	 *  set has been successfully synced, and will now start to
+	 *  receive periodic advertising reports.
+	 *
+	 *  @param sync The periodic advertising sync object.
+	 *  @param info Information about the sync event.
+	 */
+	void (*synced)(struct bt_le_per_adv_sync *sync,
+		       struct bt_le_per_adv_sync_synced_info *info);
+
+	/** @brief The periodic advertising sync has been terminated.
+	 *
+	 *  This callback notifies the application that the periodic advertising
+	 *  sync has been terminated, either by local request, remote request or
+	 *  because due to missing data, e.g. by being out of range or sync.
+	 *
+	 *  @param sync  The periodic advertising sync object.
+	 */
+	void (*term)(struct bt_le_per_adv_sync *sync,
+			   const struct bt_le_per_adv_sync_term_info *info);
+
+	/** @brief Periodic advertising data received.
+	 *
+	 *  This callback notifies the application of an periodic advertising
+	 *  report.
+	 *
+	 *  @param sync  The advertising set object.
+	 *  @param info  Information about the periodic advertising event.
+	 *  @param buf   Buffer containing the periodic advertising data.
+	 */
+	void (*recv)(struct bt_le_per_adv_sync *sync,
+		     const struct bt_le_per_adv_sync_recv_info *info,
+		     struct net_buf_simple *buf);
+};
+
+/** Periodic advertising sync options */
+enum {
+	/** Convenience value when no options are specified. */
+	BT_LE_PER_ADV_SYNC_OPT_NONE = 0,
+
+	/** @brief Use the periodic advertising list to sync with advertiser
+	 *
+	 *  When this option is set, the address and SID of the parameters
+	 *  are ignored.
+	*/
+	BT_LE_PER_ADV_SYNC_OPT_USE_PER_ADV_LIST = BIT(0),
+
+	/** @brief Disables periodic advertising reports
+	 *
+	 *  No advertisement reports will be handled until enabled.
+	 */
+	BT_LE_PER_ADV_SYNC_OPT_REPORTING_INITIALLY_DISABLED = BIT(1),
+
+	/** Sync with Angle of Arrival (AoA) constant tone extension */
+	BT_LE_PER_ADV_SYNC_OPT_DONT_SYNC_AOA = BIT(2),
+
+	/** Sync with Angle of Departure (AoD) 1us constant tone extension */
+	BT_LE_PER_ADV_SYNC_OPT_DONT_SYNC_AOD_1US = BIT(3),
+
+	/** Sync with Angle of Departure (AoD) 2us constant tone extension */
+	BT_LE_PER_ADV_SYNC_OPT_DONT_SYNC_AOD_2US = BIT(4),
+
+	/** Do not sync to packets without a constant tone extension */
+	BT_LE_PER_ADV_SYNC_OPT_SYNC_ONLY_CONST_TONE_EXT = BIT(5),
+};
+
+struct bt_le_per_adv_sync_param {
+	/** @brief Periodic Advertiser Address
+	 *
+	 *  Only valid if not using the periodic advertising list
+	 */
+	bt_addr_le_t addr;
+
+	/** @brief Advertiser SID
+	 *
+	 *  Only valid if not using the periodic advertising list
+	 */
+	uint8_t sid;
+
+	/** Bit-field of periodic advertising sync options. */
+	uint32_t options;
+
+	/** @brief Maximum event skip
+	 *
+	 *  Maximum number of periodic advertising events that can be
+	 *  skipped after a successful receive
+	 */
+	uint16_t skip;
+
+	/** @brief Synchronization timeout (N * 10 MS)
```

Comment:
The unit of `ms` is done differently in many places. I think the correct way should be `N * 10 ms`
This PR uses a mix of uppercase and lowercase, and there should be a space between the number and the unit.